SMS19/urls.py

```diff
@@ -16,21 +16,17 @@
 from django.contrib import admin
 from django.urls import path, include
 from main import views
+from django.conf.urls import url
 
 
 urlpatterns = [
 
```

Comment:
The urls that were handling authentication earlier and were linked to the authentication functions in views.py are removed . We don't need them . The include('auth.urls') provides for all authentication urls mentioned in urls.md in this PR